pset-functions.js

```diff
@@ -136,8 +164,40 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const word = "function"
+const scoreKey = {a: 1, b: 3, c: 3, d: 2, e: 1, f: 4, g: 2, h: 4, i: 1, j: 8, k: 5, l: 1, m: 3, n: 1, o: 1, p: 3, q: 10, r: 1, s: 1, t: 1, u: 1, v: 4, w: 4, x: 8, y: 4, z: 10}
+
+function scrabbleScore(word) {
+
+
+const wordArray = word.split ('')
+let score = 0
+
+wordArray.forEach(letter => {
+
+score = score + scoreKey[letter]
+});
+
+ return score
+
+}
+console.log(scrabbleScore("function"))
+
+  }
+
+
+
+const sum = [...word].reduce((accu, letter) => { return accu + scrabble[letter.toLowerCase()]; }, 0);
```

Comment:
@Ashbk32 seems that you have a syntax error. Looks like you may have tried 2 solutions here. The first approach looks to be on the right path. Take a look at the solution for an alternative approach (it's slightly different than the approach you took, but you can get a sense of how another developer approached this problem): https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L178